src/ostorlab/cli/serve/serve.py

```diff
@@ -19,6 +21,11 @@ def serve(ctx: click.core.Context, host: str, port: int, refresh_api_key: bool)
     """Run the Flask serve with the specified host and port."""
     try:
         from ostorlab.serve_app import app
+        import werkzeug
+
+        cli = sys.modules["flask.cli"]
+        cli.show_server_banner = lambda *x: None
+        werkzeug.serving.BaseWSGIServer.log_startup = lambda *x: None
```

Comment:
Seems like dirty trick! are you sure this is the right way to do it!!

Reply:
do you have a better way to do it ?

Reply:
I do, override the BaseWSGIServer, override the method and use the new instance. This is indeed a not so great hack.

Reply:
We don't use it explicitly, its used by flask.